clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesClientApiAdapter.groovy

```diff
@@ -36,12 +37,14 @@ class KubernetesClientApiAdapter {
   static final long RETRY_MAX_WAIT_MILLIS = TimeUnit.SECONDS.toMillis(10)
   static final long RETRY_INITIAL_WAIT_MILLIS = 100
   static final int API_CALL_TIMEOUT_SECONDS = 60
+  static final String API_CALL_RESULT_FORMAT = ""
   static final String DEPLOYMENT_ANNOTATION = "deployment.kubernetes.io"
   final Registry spectatorRegistry
   final Clock spectatorClock
   final ApiClient client
   final AppsV1beta1Api apiInstance
   final ExtensionsV1beta1Api extApi
+  final AutoscalingV1Api autoScalerInstance
```

Comment:
why is this called `autoScalerInstance`? The name doesn't follow from the purpose